docs/pages/upcoming-releases.mdx

```diff
@@ -43,6 +43,11 @@ with periodic mandatory access reviews and detailed audit trail.
 
 ### 14.0.0
 
+#### (Preview) Advanced Audit Log
+
+Teleport will include a new audit log backend that supports efficient
+searching, sorting, and filtering operations.
+
```

Comment:
@jimbishopp @tobiaszheller Do you think you'll have the self-hosted version of the Advanced Audit Log ready for Teleport 14 (September 2023).